src/components/Play.js

```diff
@@ -195,33 +199,31 @@ const PlayInfo = ({corpusId, playId}) => {
           )}
         </p>
         <ul className="play-meta">
-          {play.authors.map(a => (
-            <li key={a.key} className="data-link-label" id="play-author">
-              {a.name}{' '}
-              {a.key && <IdLink>{a.key}</IdLink>}
-            </li>
-          ))}
-          <br/>
           {play.source && (
             <li>
               Source:{' '}
-              {play.source.url
-                ? <a target="_blank" rel="noopener noreferrer" href={play.source.url}>{play.source.name}</a>
-                : play.source.name
-              }
+              {play.source.url ? (
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={play.source.url}
+                >
+                  {play.source.name}
+                </a>
+              ) : (
+                play.source.name
+              )}
             </li>
           )}
 
           {play.originalSource && (
-            <li>
-              Original Source:{' '}
-              {play.originalSource}
-            </li>
+            <li>Original Source: {play.originalSource}</li>
           )}
-          <li>DraCor: <em>{play.id}</em></li>
+          <li>
+            DraCor: <em>{play.id}</em>
+          </li>
           <li className="play-downloads">
-            Downloads:
-            {' '}
+            Downloads:{' '}
```

Comment:
Let's commit changes that only reformat code without modifying the logic separately.